### PR TITLE
Add support for Issues #28

### PR DESCRIPTION
**Pull request by adambraimbridge**
Solves #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
-# GitHub Command Line Tool
+# 🦑 ika
```
Comment (adambraimbridge) [inline README.md]: I messaged the owner of https://www.npmjs.com/package/ika, asking for the package name. Waiting for a reply.

File: src/commands/issues/list.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+			result.forEach(issue => {
+				const { title, created_at, html_url, assignees } = issue
+				const assigneeString = assignees.map(user => user.login).join(', ')
+				console.log(
+					// Prettier-ignore
+					`\x1b[36m${created_at}\x1b[0m`,
+					title,
+					`\x1b[33m${assigneeString || 'unassigned'}\x1b[0m`,
+					`\x1b[2m${html_url}\x1b[0m`
+				)
```
Comment (adambraimbridge) [inline src/commands/issues/list.js]: The noise above is how to add color to console strings. I could use `chalk`, but it's working nicely as-is.

File: README.md (hunk 1 above)
Comment (adambraimbridge) [reply README.md]: If the reply is "no", I'll rename to something else.

**Comment by taktran** [comment]
Does this PR need some rebasing? Seems like it has the `ika` name change that appears to already be in the `master` branch.

**Comment by adambraimbridge** [comment]
Good spot, thanks

**Pull request closed without merging**